docs/v1.1.0/schema.md

```diff
@@ -0,0 +1,216 @@
+
+<a name="dataset_list"><sup>**\[1\]**</sup></a> In case of multiple entries, they MUST be specified as a string containing a comma-separated list.
+
+### 2. Sample and experiment conditions
+Describe the sample (e.g. sample type, organism, cell type) and experiment condition (including sample preparation and other experimental conditions such as microscopy setup) for all tilt experiments in the dataset
+
+SampleConditions is a (nested) dictionary. If a data element has sub-elements, it is a dictionary.
+
+| Data elements       | metadata key 1     | Sub-elements             | metadata key 2 | Value type      | Unit   | Default  | Requirement | Controlled vocabularies                                                                                                        | Description                                                                                                                                                                                         |
+|---------------------|--------------------|--------------------------|----------------|-----------------|--------|----------|-------------|--------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Sample type         | sample_type        | N/A                      | N/A            | ENUMERATION[^1] | N/A    | N/A      | MUST        | Cell, tissue, organism, Intact organelle, In-vitro mixture of macromolecules or their complex, In-silico synthetic data, Other | Type of samples used in a CryoET study.                                                                                                                                                             |
+| Organism            | organism           | Organism name            | name           | STRING[^2]      | N/A    | N/A      | RECOMMENDED |                                                                                                                                | Name of the organism from which a biological sample used in a CryoET study is derived from, e.g. homo sapiens (reference)                                                                           |
+|                     |                    | Taxonomy ID              | taxonomy_id    | STRING          | N/A    | N/A      | RECOMMENDED | NCBI taxonomy ID                                                                                                               | NCBI taxonomy identifier for the organism, e.g. 9606 (reference)                                                                                                                                    |
```

Comment:
Would be good to actually be consistent and use the [NCBITaxon](https://obofoundry.org/ontology/ncbitaxon.html) ontology instead of straight IDs, to be consistent across the database.